parcels/fieldset.py

```diff
@@ -723,11 +724,12 @@ def from_croco(
     ):
         """Initialises FieldSet object from NetCDF files of CROCO fields.
         All parameters and keywords are exactly the same as for FieldSet.from_nemo(), except that
-        the vertical coordinate is scaled by the bathymetry (``h``) field from CROCO, in order to
-        account for the sigma-grid. The horizontal interpolation uses the MITgcm grid indexing
-        as described in FieldSet.from_mitgcm().
+        the vertical coordinate is scaled by the bathymetry (``h``) and sea-surface height (``zeta``)
+        fields from CROCO, in order to account for the sigma-grid.
+        The horizontal interpolation uses the MITgcm grid indexing as described in FieldSet.from_mitgcm().
 
-        The sigma grid scaling means that FieldSet.from_croco() requires a variable ``H: h`` to work.
+        In 3D, when there is a ``depth`` dimension, the sigma grid scaling means that FieldSet.from_croco()
+        requires variables ``H: h`` and ``Zeta: zeta``, as well as he ``hc`` parameter to work.
 
         See `the CROCO 3D tutorial <../examples/tutorial_croco_3D.ipynb>`__ for more infomation.
         """
```

Comment:
Should we add info here on the `hc` param and what it is?

Reply:
Thanks, I just clarified the doctoring a bit more